test/e2e/empty_dir.go

```diff
@@ -37,8 +37,7 @@ var _ = Describe("EmptyDir volumes", func() {
 
 	f := NewFramework("emptydir")
 
-	// TODO: Remove [Skipped] when security context is enabled everywhere
-	Context("when FSGroup is specified [Skipped]", func() {
+	Context("when FSGroup is specified [Feature:FSGroup]", func() {
```

Comment:
the specs in here will be tagged with both [Feature] and [Conformance] - is that intended? seems to be at odds with the docs.